src/components/Table/Table/Table.tsx

```diff
@@ -1,18 +1,22 @@
 import React, { FC, ReactElement } from "react";
 import cx from "classnames";
-import { VibeComponentProps } from "../../../types";
+import { SubIcon, VibeComponentProps } from "../../../types";
 import styles from "./Table.module.scss";
 import { ITableHeaderProps } from "../TableHeader/TableHeader";
 import { ITableBodyProps } from "../TableBody/TableBody";
 import { getTableRowLayoutStyles } from "./tableHelpers";
 import { getTestId } from "../../../tests/test-ids-utils";
 import { ComponentDefaultTestId } from "../../../tests/constants";
 
+export type TableContentSkeletonType = "long-text" | "medium-text" | "circle" | "rectangle";
+
 export interface ITableColumn {
   id: string;
   title: string;
   infoContent?: string;
   width?: number | { min: number; max: number };
+  icon?: SubIcon;
+  contentSkeleton?: TableContentSkeletonType | TableContentSkeletonType[];
```

Comment:
name-wize, I'd use something that indicates `loading`. the fact that it's currently a skeleton doesn't really matter.
I suggested `loadingStateType` in the [API RFC](https://github.com/mondaycom/monday-ui-react-core/issues/1471)

Reply:
wasn't aware of the RFC